Automattic/vip-go-mu-plugins
# Add user last seen information

## Description

This PR introduces a `Last Seen` user meta to the users, which is going to be used to report and block inactive users, improving security by reducing the surface attack.

## Inactive users blocked on login
<img width="40%" src="https://github.com/Automattic/vip-go-mu-plugins/assets/33168/a5cd16c4-057b-49e8-9dd0-92c11fd9db02">

## Inactive users on the Users page (network and site)
<img src="https://github.com/Automattic/vip-go-mu-plugins/assets/33168/178a4f37-4054-4748-8b04-e4229caf1bbc">

## Inactive users 
```
curl -X GET --user "example@example.org:key" "http://vip-testing-luismulinari-multi.vipdev.lndo.site/wp-json/wp/v2/users?context=edit"
{"code":"rest_forbidden_context","message":"Sorry, you are not allowed to list users.","data":{"status":401}}%
```


## Changelog Description

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test
1. Check out PR.
2. Add the following constants:
```
define( 'VIP_SECURITY_INACTIVE_USERS_ACTION', 'BLOCK' );
define( 'VIP_SECURITY_CONSIDER_USERS_INACTIVE_AFTER_DAYS', 30 );
```
3. Go to `wp-admin` > `Users`
